src/modules/auth/api.js

```diff
@@ -10,26 +10,49 @@ export class SeaCatAuthApi {
 
 	module.exports = {
 		app: {
-			"OIDC_URL": 'http://localhost:3000/openidconnect',
+			BASE_URL: 'http://localhost:3000',
+			microservice: '/api',
+			subpaths: {oidc: '/openidconnect', rbac: '/rbac'},
 			...
 	*/
 
 	constructor(config) {
-		
-		this.BaseURL = config.get('OIDC_URL');
+
+		this.BaseURL = config.get('BASE_URL');
+		this.microservice = config.get('microservice');
```

Comment:
`this.Microservice` ... please be consistent - applies to other member variables too.